README.md

```diff
@@ -1,16 +1,144 @@
-
-- a `README.md` outlining the project goals, sponsoring sig, and community contact information
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+# kubectl-validate
+
+kubectl-validate is a SIG-CLI subproject to support the local validation of 
+resources for native Kubernetes types and CRDs.
+
+This project has two goals:
+
+1.) Shift-left validation of resources with as close to parity to server-side
+Kubernetes as possible.
+2.) Improve declarative validation support in upstream Kubernetes over time,
+making those improvements available for kubectl-validate users early.
+
+## Comparison With Other Tools
+
+`kubectl-validate` distinguishes itself among other OpenAPI-based validators by
+to its deep integration with Kubernetes upstream validation. `kubectl-validate`
+is written by Kubernetes apiserver authors using the same code as the server-side.
+
+This allows `kubectl-validate` to give the most accurate error messages and support
+the Kubernetes-specific validations often ignored by other tools.
+
+## Drawbacks
+
+`kubectl-validate` suffers from similar drawbacks to other OpenAPI-based validation tools.
+For CRDs, users should expect a first class experience likely identical to how the server treats CRDs.
+
+For Native Types, the OpenAPI definitions are a best-effort replication of the
+handwritten validation rules conducted by the apiserver. There is an ongoing 
+effort to improve the quality of native type definitions, and eventually replace 
+the handwritten rules. As the improvements are made upstream, kubectl-validate 
+will integrate them to make them available to all users.
+
+Bottom line is that users should expect nearly a seamless experience with CRDs,
+and great but constantly improving support for the builtin Kubernetes types.
+
+# Installation
+
+You can get started using kubectl-validate right away with an existing installation
+of Go:
+
+```sh
+go install sigs.k8s.io/kubectl-validate@latest
+```
+
+# Usage
+
+Once installed, you have the option of invoking `kubectl-validate` through
+`kubectl` or directly:
+
+## Example
+
+```sh
+kubectl validate ./path/to/file.yaml
```

Comment:
@seans3 How bad is it that a kubectl plugin doesn't use the (IMHO terrible) `-f <file>` syntax?